### PR TITLE
Add project files to best-practices samples

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/standard/base-types/best-practices-strings.md [hunk 1]
```diff
@@ -313,16 +313,18 @@ To explicitly specify that a string should be formatted by using the conventions
 
 - For string concatenation, do not allow the compiler to perform any implicit conversions. Instead, perform an explicit conversion by calling a `ToString` overload that has a `provider` parameter. For example, the compiler implicitly uses the current culture when converting a <xref:System.Double> value to a string in the following C# code:
```
Comment (IEvangelist) [inline docs/standard/base-types/best-practices-strings.md:314]:
```suggestion
- For string concatenation, do not allow the compiler to perform any implicit conversions. Instead, perform an explicit conversion by calling a `ToString` overload that has a `provider` parameter. For example, the compiler implicitly uses the current culture when converting a <xref:System.Double> value to a string in the following code:
```

Remove the **C#** callout here and just call it code, as you've added VB.

**Review by IEvangelist — CHANGES_REQUESTED** [review]
Please fix this one aspect of the pull request, and we should be able to continue reviewing it. Thank you

File: docs/standard/base-types/best-practices-strings.md [hunk 2]
```diff
@@ -311,18 +311,20 @@ To explicitly specify that a string should be formatted by using the conventions
...
+- For string concatenation, do not allow the compiler to perform any implicit conversions. Instead, perform an explicit conversion by calling a `ToString` overload that has a `provider` parameter. For example, the compiler implicitly uses the current culture when converting a <xref:System.Double> value to a string in the following code:
 
-  [!code-csharp[Implicit String Conversion](~/samples/snippets/standard/base-types/string-practices/cs/tostring.cs#1)]
+  [!code-csharp[Implicit String Conversion](~/samples/snippets/standard/base-types/string-practices/cs/tostring/Program.cs#1)]
+  [!code-vb[Implicit String Conversion](~/samples/snippets/standard/base-types/string-practices/vb/tostring/Program.vb#1)]
 
   Instead, you can explicitly specify the culture whose formatting conventions are used in the conversion by calling the <xref:System.Double.ToString(System.IFormatProvider)?displayProperty=nameWithType> method, as the following C# code does:
```
Comment (Youssef1313) [inline docs/standard/base-types/best-practices-strings.md:319]:
```suggestion
  Instead, you can explicitly specify the culture whose formatting conventions are used in the conversion by calling the <xref:System.Double.ToString(System.IFormatProvider)?displayProperty=nameWithType> method, as the following code does:
```

**Comment by Thraka** [comment]
This looks like an opportunity to move the snippet out of the samples folder and into a snippets folder with the article.

**Comment by Youssef1313** [comment]
> 
> 
> This looks like an opportunity to move the snippet out of the samples folder and into a snippets folder with the article.
> 
> https://docs.microsoft.com/en-us/contribute/dotnet/dotnet-contribute#example-structure

I've moved them

**Review by IEvangelist — APPROVED** [review]
Looks good to me :shipit:

**Pull request merged**